pkg/app/ops/mysqlensurer/indexes.sql

```diff
@@ -6,70 +6,70 @@
 CREATE INDEX application_disabled_updated_at_desc ON Application (Disabled, UpdatedAt DESC);
 
 -- index on `EnvId` ASC and `UpdatedAt` DESC
-ALTER TABLE Application ADD COLUMN EnvId VARCHAR(36) GENERATED ALWAYS AS (data->>"$.env_id") VIRTUAL;
+ALTER TABLE Application ADD COLUMN EnvId VARCHAR(36) GENERATED ALWAYS AS (data->>"$.env_id") VIRTUAL NOT NULL;
 CREATE INDEX application_env_id_updated_at_desc ON Application (EnvId, UpdatedAt DESC);
 
 -- index on `Name` ASC and `UpdatedAt` DESC
-ALTER TABLE Application ADD COLUMN Name VARCHAR(50) GENERATED ALWAYS AS (data->>"$.name") VIRTUAL;
+ALTER TABLE Application ADD COLUMN Name VARCHAR(50) GENERATED ALWAYS AS (data->>"$.name") VIRTUAL NOT NULL;
 CREATE INDEX application_name_updated_at_desc ON Application (Name, UpdatedAt DESC);
 
 -- index on `Deleted` and `CreatedAt` ASC
 -- TODO: Reconsider make this Deleted column as STORED GENERATED COLUMN
-ALTER TABLE Application ADD COLUMN Deleted BOOL GENERATED ALWAYS AS (data->>"$.deleted") VIRTUAL;
+ALTER TABLE Application ADD COLUMN Deleted BOOL GENERATED ALWAYS AS (IFNULL(data->>"$.deleted", False)) VIRTUAL NOT NULL;
 CREATE INDEX application_deleted_created_at_asc ON Application (Deleted, CreatedAt);
 
 -- index on `Kind` ASC and `UpdatedAt` DESC
-ALTER TABLE Application ADD COLUMN Kind INT GENERATED ALWAYS AS (data->>"$.kind") VIRTUAL;
+ALTER TABLE Application ADD COLUMN Kind INT GENERATED ALWAYS AS (data->>"$.kind") VIRTUAL NOT NULL;
```

Comment:
Not sure why the default value for Kind is not specified?

Reply:
Thanks for comment 🙏 I made commit to add default value for this `kind` 👍